sources/packages/web/src/components/common/AwardTable.vue

```diff
@@ -48,10 +48,15 @@ export default defineComponent({
     );
 
     const getAwardValue = (awardType: string): string | number => {
-      return (
-        props.awardDetails[`${props.identifier}${awardType.toLowerCase()}`] ??
-        "(Not eligible)"
-      );
+      const awardValue =
+        props.awardDetails[`${props.identifier}${awardType.toLowerCase()}`];
+      // If the award is defined but no values are present it means that a receipt value is missing.
+      if (awardValue === null) {
+        return "-";
+      }
+      // If the award in not defined at all it means that the award is not eligible an it was not
```

Comment:
If the award in not defined at all it means that the award is not eligible **and** it was not

Reply:
Changed.